src/data/cache.ts

```diff
@@ -163,6 +168,11 @@ export const cache: InMemoryCache = new InMemoryCache({
             };
           },
         },
+        articles: createReactiveField({
```

Comment:
I'm trying to look up the way to draw these variables closer to the context, but I'm starting to doubt there's one, given the API direction Apolo takes with the new reactive variable hook :(